### ISSUE TITLE
Add support for importing aws_ecs_task_definition

### ISSUE THREAD

**Issue by wolfgangmeyers** [terraform-providers/terraform-provider-aws#5008]
Hello,

I searched through issues in both terraform and terraform-providers and didn't find any that brought this up. Is it feasible to add support for importing aws_ecs_task_definition? It doesn't seem to be supported on the version that I have:

```
$ terraform -version
Terraform v0.11.7
+ provider.aws v1.24.0
```

I tried to import a hello-world type task definition and was informed that this is currently unsupported:
```
$ terraform import aws_ecs_task_definition.fargate-email fargate-email
provider.aws.region
  The region where AWS operations will take place. Examples
  are us-east-1, us-west-2, etc.

  Default: us-east-1
  Enter a value:

aws_ecs_task_definition.fargate-email: Importing from ID "fargate-email"...

Error: aws_ecs_task_definition.fargate-email (import id: fargate-email): import aws_ecs_task_definition.fargate-email (id: fargate-email): resource aws_ecs_task_definition doesn't support import
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* aws_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by wolfgangmeyers** [comment]
Also, I'm comfortable with go development so if anyone could point me in the right direction for this I might be able to contribute 😸

**Comment by tomelliff** [comment]
@wolfgangmeyers if you're feeling up to contributing this feature you should be able to largely follow what has been done with other import support additions such as for [ECR repositories](https://github.com/terraform-providers/terraform-provider-aws/commit/2f4bc9df439bc6983c417e8cad6f22eb251e5c4d).

The [`schema.ImportStatePassthrough`](https://github.com/hashicorp/terraform/blob/7a29a03e2179c16d11da053228aa76677a78f582/helper/schema/resource_importer.go#L47-L52) method provides a convenient way to import a resource based on its ID/ARN as long as that resource can be read that way. Thankfully that seems to be the case here: https://github.com/terraform-providers/terraform-provider-aws/blob/25655c2b04b8f354701d3a0c99ce36c21949de92/aws/resource_aws_ecs_task_definition.go#L241-L243

You'll need to add an acceptance test that piggy backs off the other acceptance tests to create a task definition and then check that it imports the state properly as you can see in the above linked commit. After that you should then run the acceptance test(s) locally and show the output in your PR. I also like to build the provider locally and then use it directly so I can play around with edge cases/get real world usage.

Your PR should also include adding a section to the docs so other users now know that they can import ECS task definitions.

Let me know if you need any more pointers :)

**Comment by bflad** [comment]
Support for this has been merged in and will release with version 1.51.0 of the AWS provider, likely later today. 👍
